packages/@apphosting/adapter-nextjs/src/bin/build.ts

```diff
@@ -50,21 +48,22 @@ const generateBundleYaml = async () => {
     const redirects = manifest.redirects.filter(it => !it.internal).map(it => ({...it, regex: undefined}));
     const beforeFileRewrites = Array.isArray(manifest.rewrites) ? manifest.rewrites : manifest.rewrites?.beforeFiles || [];
     const rewrites = beforeFileRewrites.map(it => ({...it, regex: undefined}));
+    const outputBundleDirectory = dirname(outputBundlePath);
     await writeFile(outputBundlePath, yamlStringify({
         headers, 
         redirects, 
         rewrites,
-        runCommand: `node ${serverFilePath}`,
-        neededDirs: [appHostingOutputDirectory],
-        staticAssets: [appHostingStaticDirectory, appHostingPublicDirectory],
+        runCommand: `node ${normalize(relative(outputBundleDirectory, serverFilePath))}`,
+        neededDirs: [normalize(relative(outputBundleDirectory, appHostingOutputDirectory))],
+        staticAssets: [normalize(relative(outputBundleDirectory, appHostingPublicDirectory))],
```

Comment:
i think this is missing the static directory

Reply:
static is now part of .next directory per NextJS standalone specifications, we can further optimize when we have CDN. SG?

Reply:
SG 👍